input.go

```diff
@@ -122,9 +122,13 @@ type Touch interface {
 
 // Touches returns the current touch states.
 //
+// Touches will return nil when there are no touches.
 // Touches always returns nil on desktops.
 func Touches() []Touch {
 	t := ui.AdjustedTouches()
+	if len(t) == 0 {
+		return nil
+	}
```

Comment:
Another way to avoid `if` clause would be:

```go
ts := ui.AdjustedTouches()
var ts2 []Touch
for _, t := range ts {
    ts2 = append(ts2, t)
}
return ts2
```